.pipelines/mac-ios-spm-release-validation-pipeline.yml

```diff
@@ -14,10 +14,40 @@
     - template: templates/use-xcode-version.yml
       parameters:
         xcodeVersion: ${{ variables.xcodeVersion }}
+    
+    # Note: Currently it requires a dev version extensions c pod for testing
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        buildType: 'specific'
+        project: 'Lotus'
+        definition: 1206          #'definitionid' is obtained from `System.DefinitionId` of extensions CI: extensions.ios_packaging
+        buildVersionToDownload: 'specific'
+        branchName: 'yguo/change-header-path'   # TODO: update branch name here
+        pipelineID: 331190
+        targetPath: '$(Build.ArtifactStagingDirectory)'
 
     - script: |
         set -e -x
-        xcodebuild test -scheme onnxruntime -destination 'platform=iOS Simulator,name=iPhone 14'
+        ls
+      workingDirectory: '$(Build.ArtifactStagingDirectory)/ios_packaging_artifacts'
+      displayName: "List staged artifacts for ORT Ext C Pod"
```

Comment:
nit: indentation looks inconsistent

Reply:
ah sorry - indentation is correct - it's the script body that is indented further than the following parameters